src/modules/navigation/ItemNavigation.tsx

```diff
@@ -44,13 +56,20 @@ const DrawerNavigation = (): JSX.Element | null => {
     return <LoadingTree />;
   }
 
+  if (shuffle) {
+    const baseId = rootId || '';
+    const memberId = member?.id || '';
+    const combinedUuids = combineUuids(baseId, memberId);
```

Comment:
is it important to combine the item id with the member id, is it not enough to use the member id ?

Reply:
@spaenleh, you have to. Otherwise, since our shuffling mechanism just switches the indexes in an array (Fisher-Yates), for all questionnaires of N length, you would always have the same order.